config/database.yml

```diff
@@ -6,7 +6,7 @@ default: &default
 
 development:
   <<: *default
-  database: pecs_move_platform_backend_development
+  database: pecs_staging
```

Comment:
Accidentally committed?

Reply:
Oh yes, that's a good catch. Will revert.